docs/paper/admin/reference/configuration/bukkit-configuration.mdx

```diff
@@ -0,0 +1,122 @@
+---
+slug: /reference/bukkit-configuration
+---
+
+# Bukkit Configuration
+
+import React from 'react';
+import Config from '@site/src/components/Config';
+
+:::info
+
+The below YAML shows you the structure and default values for `bukkit.yml`
+
+Click on a leaf node to view the description for that setting.
+
+:::
+
+<Config data={`
+aliases:
+  default: 'now-in-commands.yml'
+  description: 'Static value from converting old versions to the new commands.yml format.'
+chunk-gc:
+  period-in-ticks:
+    default: '600'
+    description: 'How long chunks loaded by plugins should last for.
+      Capped by paper to be 20 ticks (1 second).'
+settings:
+  allow-end:
+    default: 'true'
+    description: 'Whether to load end-type dimensions.'
+  connection-throttle:
+    default: '4000'
+    description: 'How long of a delay to enforce between connections from an IP address.
+      Measured in milliseconds since last attempt.'
+  deprecated-verbose:
+    default: 'default'
+    description: 'Whether to warn for use of deprecated events.
+      If "default", the warning is printed if annotated with \`@Warning(true)\`.
+      Ignored if [system property paper.alwaysPrintWarningState](/paper/reference/system-properties#paperalwaysprintwarningstate) is true.
+
+      Valid values are "true", "false", or "default".'
+  minimum-api:
+    default: 'none'
+    description: 'Minimum plugin api-version. If below this, or not specified, the plugin is prevented from loading.'
+  permissions-file:
+    default: 'permissions.yml'
+    description: 'File to load server permissions from.'
```

Comment:
Maybe this should explain how it works, or link to a page explaining it. And that it's usually not used anymore (to my knowledge?)